karura/integration/protocol-info.md

```diff
@@ -30,6 +30,18 @@ ED of native token KAR is configured in the runtime. Non-native tokens \(KSM, kU
 
 Read more on ED [here](https://github.com/AcalaNetwork/Acala/wiki/A.-Existential-Deposit).
 
+## Protocol Fees
+
+- **DeX swap:** 22 milliKAR~
```

Comment:
the KAR fees are platform fee, not protocol fee, please exclude them here

Reply:
@xlc Just notice your comment.  Thanks.